src/transforms/group.js

```diff
@@ -164,6 +164,7 @@ export function maybeSubgroup(outputs, Z, F, S) {
 
 function reduceFunction(f) {
   return {
+    label: f.label,
```

Comment:
This change seems unrelated (and untested), so I’m going to revert this part. Elsewhere we don’t promote the reducer name to the axis label so I think it probably isn’t appropriate to promote it here. (For example, when you use the “sum” reducer, we don’t use or want “sum” as the axis label; we want the name of the column being summed.)